<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUIPrintCheck</class>
 <widget class="QWidget" name="MainGUIPrintCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1586</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainGUIPrintCheck</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="frameDashBoardHoseBeginning">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 40px;
     max-height: 40px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QRadioButton{
     background:rgb(255,255,255);
      color: rgb(39,51,66);
     font-size: 16px;
     font-weight: bold;
     font-family: Siemens Sans;
     min-height: 32px;
     max-height: 32px;
}

QRadioButton::indicator {
width: 15px;
height: 15px;
border-radius: 15px;
}

QRadioButton::indicator::unchecked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: white; 
width: 13px; 
height: 13px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: rgb(0,95,135); 
width: 13px; 
height:13px; 
}



</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_21">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonHoseBeginning">
          <property name="text">
           <string>Hose Beginning</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QFrame" name="frameDashBoardHoseMiddle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 40px;
     max-height: 40px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}
QRadioButton{
     background:rgb(255,255,255);
      color: rgb(39,51,66);
     font-size: 16px;
     font-weight: bold;
     font-family: Siemens Sans;
     min-height: 32px;
     max-height: 32px;
}

QRadioButton::indicator {
width: 15px;
height: 15px;
border-radius: 15px;
}

QRadioButton::indicator::unchecked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: white; 
width: 13px; 
height: 13px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: rgb(0,95,135); 
width: 13px; 
height:13px; 
}


</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_22">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonHoseMiddle">
          <property name="text">
           <string>Hose Middle</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QFrame" name="frameDashBoardHoseEnd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 40px;
     max-height: 40px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}
QRadioButton{
     background:rgb(255,255,255);
      color: rgb(39,51,66);
     font-size: 16px;
     font-weight: bold;
     font-family: Siemens Sans;
     min-height: 32px;
     max-height: 32px;
}

QRadioButton::indicator {
width: 15px;
height: 15px;
border-radius: 15px;
}

QRadioButton::indicator::unchecked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: white; 
width: 13px; 
height: 13px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: rgb(0,95,135); 
width: 13px; 
height:13px; 
}



</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_23">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButtonHoseEnd">
          <property name="text">
           <string>Hose End</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QFrame" name="frameDashBoardNumberHose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>44</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 42px;
     max-height: 42px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardNumberHose">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Number Hose:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxNumberHose">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="8">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QFrame" name="frameDashBoardNumberErrorHose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>356</width>
         <height>44</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 42px;
     max-height: 42px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardNumberErrorHose">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Number Error Hose:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxNumberErrorHose">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="pushButtonResetHoseCounter">
       <property name="text">
        <string>Reset Counter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCurretDateTime">
       <property name="font">
        <font>
         <family>Siemens Sans</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <widget class="QFrame" name="frameLiveImageDataTop">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>256</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frameLiveImageDataBot">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>256</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>328</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditErrorMessage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>72</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>72</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frameDashBoardResetLongTimeError">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>74</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>74</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
     background:yellow;
     min-height: 72px;
     max-height: 72px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QRadioButton{
     background:rgb(255,255,255);
      color: rgb(39,51,66);
     font-size: 16px;
     font-weight: bold;
     font-family: Siemens Sans;
     min-height: 32px;
     max-height: 32px;
}

QRadioButton::indicator {
width: 15px;
height: 15px;
border-radius: 15px;
}

QRadioButton::indicator::unchecked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: white; 
width: 13px; 
height: 13px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: rgb(0,95,135); 
width: 13px; 
height:13px; 
}



</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_24">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonResetLongTimeErrorText">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>107</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>verticalSpacer_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>radioButtonHoseBeginning</tabstop>
  <tabstop>doubleSpinBoxNumberErrorHose</tabstop>
  <tabstop>pushButtonResetHoseCounter</tabstop>
  <tabstop>radioButtonHoseMiddle</tabstop>
  <tabstop>radioButtonHoseEnd</tabstop>
  <tabstop>doubleSpinBoxNumberHose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
